content/docs/join_us/recruitment_FAQs.md

```diff
@@ -65,17 +90,43 @@ More information can be found on [Mathison](https://mathison.turing.ac.uk/Intera
 
 Some of the team have also been featured in project and institute spotlight interviews and the [Turing podcast](https://www.podbean.com/media/share/pb-vrci3-1366a21?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share) (including episodes on [AI for the Skies](https://www.podbean.com/media/share/pb-zzidx-d7bb24?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share), [Fantasy Football AI](https://www.podbean.com/media/share/pb-5afja-df2052?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share), [3D printed bridges](https://www.podbean.com/media/share/pb-qekh3-dfe740?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share), [Reproducible data science](https://www.podbean.com/media/share/pb-wxkjp-ecfa7f?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share), [The problems of AI](https://www.podbean.com/media/share/pb-dyq36-112350b?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share) and [Living with Machines](https://www.podbean.com/media/share/pb-s9das-1275eb2?utm_campaign=w_share_ep&utm_medium=dlink&utm_source=w_share)).
 
-### What is a Research Computing Engineer?
+### I don't have a physics / maths / computer science background. Is that okay?
+
+That is completely fine.
+Our team includes people with backgrounds in chemistry and biology, as well as the social sciences.
+
+In practice, we do not often work on projects that align closely with our own academic backgrounds.
+As long as you are willing to pick up new skills and domains as you go along, your own background will not be a barrier.
+
+### I'm a woman / person of colour / LGBTQIA+ / person with disabilities / part of an underrepresented group(s). Will I feel welcome and supported in the team?
 
-This role presents a new exciting opportunity to join the growing team of research computing experts at The Turing. The team assists the Turing community with designing, building, deploying, and maintaining research infrastructures, as well as providing service delivery and user support for the Institute’s research computing platforms. This is an opportunity for a technically-skilled professional with a good understanding of either cloud or high-performance computing (HPC), experience with both application and infrastructure, and an interest in facilitating and enhancing world-leading data science and AI research.
-You will build tools and automation to eliminate mundane operations and craft repeatable processes. As part of the team, you will also work with other research software engineers and data scientists on various projects and will participate in all phases of cloud / HPC development, from requirements-capture, definition, and design; to development, deployment, and maintenance; to performance tuning and monitoring. You are also expected to keep up-to-date with the latest cutting-edge developments, and/or make use of the UK’s most advanced computing facilities like Dawn, ARCHER 2, JADE 2, and Baskerville.
+**We want you**.
```

Comment:
I'd be inclined to change this to "We'd love to have you join our team", but that's personal preference